homeassistant/components/modbus/const.py

```diff
@@ -16,13 +16,8 @@
 CONF_BYTESIZE = "bytesize"
 CONF_CLIMATES = "climates"
 CONF_CLOSE_COMM_ON_ERROR = "close_comm_on_error"
-CONF_COILS = "coils"
-CONF_CURRENT_TEMP = "current_temp_register"
-CONF_CURRENT_TEMP_REGISTER_TYPE = "current_temp_register_type"
 CONF_DATA_TYPE = "data_type"
 CONF_FANS = "fans"
-CONF_HUB = "hub"
```

Comment:
Apparently CONF_HUB was still used in https://github.com/home-assistant/core/blob/dev/homeassistant/components/stiebel_eltron/__init__.py and https://github.com/home-assistant/core/blob/dev/homeassistant/components/flexit/climate.py